ui/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './Header.css';
+
+/* Profile photo link  be passed in from the Page component */
+const Header = (props) => {
+    return(
+        <div className="header">
```

Comment:
This inline "require" isn't 100% obvious and will be dfficult to deal with later. You should move it up as an "import" at the top of the file: